package.json

```diff
@@ -5,14 +5,17 @@
   "main": "index.js",
   "dependencies": {
     "@material/button": "^2.3.0",
+    "@material/dialog": "^3.0.0",
     "@material/drawer": "^2.3.0",
+    "@material/icon-button": "^3.0.0",
     "@material/ripple": "^2.3.0",
     "@material/textfield": "^2.3.1",
     "@material/top-app-bar": "^2.3.0",
     "corejs-typeahead": "^1.2.1",
     "install": "^0.12.2",
     "jquery": "^3.4.1",
-    "material-components-web": "^2.3.1"
+    "material-components-web": "^2.3.1",
+    "material-design-icons": "file:material-design-icons"
```

Comment:
Not sure if this has been used in your changes. Please double check. If required, then leave it in, otherwise remove.

Reply:
Done, it wasn't needed.

Reply:
has this been pushed?